service/src/main/java/uk/gov/hmcts/reform/fpl/service/document/aggregator/BundleViewAggregator.java

```diff
@@ -53,4 +94,40 @@ public List<DocumentContainerView> getDocumentBundleViews(
         return bundles;
     }
 
+    private DocumentView toConfidentialDocumentView(DocumentType documentType, WithDocument md,
+                                                    ConfidentialLevel level) {
+        String filename = Optional.ofNullable(md.getDocument()).orElse(DocumentReference.builder().build())
+            .getFilename();
+        return DocumentView.builder()
+            .title(filename)
+            .fileName(filename)
+            .confidentialLevel(level == CTSC ? "Restrict to HMCTS staff"
+                : "Restrict to the LA, Cafcass and HMCTS staff")
+            .uploaderType(md.getUploaderType() == null ? "unknown" : md.getUploaderType().name())
+            .documentType(StringUtils.replace(documentType.getDescription(), "└─ ",  ""))
+            .build();
+    }
+
+    public List<DocumentView> getConfidentialDocumentBundleViews(CaseData caseData, DocumentViewType view) {
+        DocumentType[] documentTypes = new DocumentType[] {CASE_SUMMARY, POSITION_STATEMENTS, POSITION_STATEMENTS_CHILD,
```

Comment:
would DocumentType.values() give us this?

Reply:
The confidential document tab is a temporary solution. According to Sara's email, if the confidential folder was introduced, those document types would be removed from the view.

And not all the document types are fitted to the generic logic (e.g. Placement Response).